website/docs/sql_dml.md

```diff
@@ -0,0 +1,189 @@
+---
+title: SQL DML
+summary: "In this page, we go will cover details on how to modify data with Hudi tables"
+toc: true
+last_modified_at: 
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+
+# Spark DML Operations
+
+The following are SparkSQL DML actions available:
+
+Insert Into
+Merge Into
+Update
+Delete
+
+You can ingest data, modify or delete data from a given hudi table using these operations. Lets go over them one by one. 
+
+Please refer to [SQL DDL](/docs/next/sql_ddl) for creating Hudi tables. This page goes over ingesting and modifying data once the
+table is created. 
+
+## Insert Into
```

Comment:
Note to reviewer:
for now, copied over contents from our quick start page. lets land the first version. we can enhance w/ more examples once we have the first cut landed.